ext/systemd/puppet.service

```diff
@@ -6,6 +6,7 @@ After=basic.target network.target puppetmaster.service
 [Service]
 EnvironmentFile=-/etc/sysconfig/puppetagent
 EnvironmentFile=-/etc/sysconfig/puppet
+EnvironmentFile=-/etc/default/puppet
```

Comment:
I feel like we should either template this, or have a separate debian file... but that's kind of terrible also